app/code/Magento/Newsletter/Model/ResourceModel/Queue/Collection.php

```diff
@@ -225,7 +225,10 @@ public function addCustomerFilter(int $customerId): Collection
             )->join(
                 ['subscriber' => $this->getTable('newsletter_subscriber')],
                 'link.subscriber_id=subscriber.subscriber_id',
-                ['subscriber_store_id' => 'subscriber.store_id']
+                [
```

Comment:
Please cover this code changes with integration test

Reply:
covered in dev/tests/integration/testsuite/Magento/Newsletter/Model/ResourceModel/Queue/CollectionTest.php